src/ui/Quickview/Quickview.ts

```diff
@@ -275,7 +276,9 @@ export class Quickview extends Component {
     const arrow = $$('div').el;
     tooltip.appendChild(arrow);
 
-    this.buildPopper(icon, tooltip, arrow);
+    if (!$$(this.element).parent('.CoveoCardActionBar')) {
```

Comment:
Few suggestions here:
1- Is there a `layout` property somewhere that holds the value of "list" vs. "card" that we could read instead of doing a dom search for a CSS class?
2- Usually when an `if` condition becomes complicated like this, I like to extract it into a variable.
```
const resultLayoutIsCard = $$(this.element).parent('.CoveoCardActionBar';

if (!resultLayoutIsCard) {
    // logic
}
```

3- Atm, we are hiding the tooltip when in card more by not calling `buildPopper`. However, maybe we should not enter `buildTooltip`? Should the method be renamed to `buildTooltipIfNotInCardLayout` for example, and we move the layout check to the top of the method?

4- Are there any CSS styles that could be cleaned up now that we are showing/hiding the tooltip with popper instead of using CSS?

Reply:
1- Done, using QueryStateModel layout property
2- Done
3- Done, I refactored some the naming which is now a lot clearer
4- Popper is only for positioning the tooltip here, not the show/hide. It is still done by css